public/app/features/explore/MetaInfoText.tsx

```diff
@@ -1,34 +1,34 @@
 import React, { memo } from 'react';
 import { css } from '@emotion/css';
-import { GrafanaTheme } from '@grafana/data';
-import { useStyles } from '@grafana/ui';
+import { GrafanaTheme2 } from '@grafana/data';
+import { useStyles2 } from '@grafana/ui';
 
-const getStyles = (theme: GrafanaTheme) => ({
+const getStyles = (theme: GrafanaTheme2) => ({
   metaContainer: css`
     flex: 1;
-    color: ${theme.colors.textWeak};
-    margin-bottom: ${theme.spacing.d};
+    color: ${theme.colors.text.secondary};
+    margin-bottom: ${theme.spacing(2)};
     min-width: 30%;
     display: flex;
     flex-wrap: wrap;
   `,
   metaItem: css`
-    margin-right: ${theme.spacing.d};
-    margin-top: ${theme.spacing.xs};
+    margin-right: ${theme.spacing(2)};
+    margin-top: ${theme.spacing(0.5)};
     display: flex;
     align-items: baseline;
 
     .logs-meta-item__error {
-      color: ${theme.palette.red};
+      color: ${theme.colors.warning.main};
```

Comment:
i think this should be `error.text` instead
```suggestion
      color: ${theme.colors.error.text};
```